src/web/components/SignedInAs.tsx

```diff
@@ -99,7 +99,7 @@ export const SignedInAs = ({
 						color: ${neutral[60]};
 					`}
 				>
-					({commentCount})
+					({commentCount || '...'})
```

Comment:
Are these three dots or an ellipsis... 😁

Reply:
... vs …

Reply:
I went with the non dot dot dot version 👍